libs/platform/src/lib/components/form/multi-input/multi-input.component.scss

```diff
@@ -0,0 +1,32 @@
+.popover-mulit-input-custom {
+    display: inline !important;
```

Comment:
you can import token/tokenizer scss file for styles.

Reply:
not necessary, i am using the token component from core.